epub34/authoring/index.html

```diff
@@ -10500,44 +10407,42 @@ <h4>The <code>prefix</code> attribute</h4>
 					<p>Note that for <a href="#sec-xhtml-svg-inclusion">SVG embedded by inclusion</a>, prefixes MUST be
 						declared on the [[html]] root [^html^] element.</p>
 
-					<p>To avoid conflicts, EPUB creators MUST NOT use the <code>prefix</code> attribute to declare a
-						prefix that maps to a <a href="#sec-default-vocab">default vocabulary</a>.</p>
+					<p>To avoid conflicts, the <code>prefix</code> attribute MUST NOT be used to declare a prefix that
+						maps to a <a href="#sec-default-vocab">default vocabulary</a>.</p>
 
-					<p>EPUB creators MUST NOT declare the prefix '_' as this specification reserves this prefix for
-						future compatibility with RDFa [[rdfa-core]] processing.</p>
+					<p>The prefix '_' MUST NOT be declared as this specification reserves this prefix for future
+						compatibility with RDFa [[rdfa-core]] processing.</p>
 
-					<p>For future compatibility with alternative serializations of the package document, EPUB creators
-						MUST NOT declare a prefix for the Dublin Core <em>/elements/1.1/</em> namespace [[dcterms]].
-						[=EPUB creators=] MUST use only the [[dcterms]] elements <a href="#sec-pkg-metadata">allowed in
-							the package document metadata</a>.</p>
+					<p>For future compatibility with alternative serializations of the package document, a prefix MUST
+						NOT be declared for the Dublin Core <em>/elements/1.1/</em> namespace [[dcterms]]. Only the
+						[[dcterms]] elements are <a href="#sec-pkg-metadata">allowed in the package document
+							metadata</a>.</p>
 				</section>
 
 				<section id="sec-reserved-prefixes">
 					<h4>Reserved prefixes</h4>
 
 					<div class="caution">
-						<p>Although reserved prefixes are an authoring convenience, [=EPUB creators=] should avoid
-							relying on them as they may cause interoperability issues. [=EPUB conformance checkers=]
-							will often reject new prefixes until their developers update the tools to the latest version
-							of the specification, for example. EPUB creators should declare all prefixes they use to
-							avoid such issues.</p>
+						<p>Although reserved prefixes are an authoring convenience, they can cause issues in the supply
+							chain. Vendors, for example, will often reject new prefixes until they update their [=EPUB
```

Comment:
```suggestion
						<p>Although reserved prefixes are an authoring convenience, they can cause interoperability issues. Vendors, for example, will often reject new prefixes until they update their [=EPUB
```

Trying to be closer to the original text...

Reply:
Vendor rejection isn't what I would call an interop problem, though. There is a disconnect between the claim and the example. The usual problem was exactly what the example cites, which is that vendors would reject epubs with new reserved prefixes because their versions of epubcheck were out of date and gave warnings.

If we want to keep the claim that it causes interop issues, we should either find a broader way to state the problem that includes supply chain problems or cite an example of a problem that reading systems have had with reserved prefixes. (I honestly can't think of a case off the top of my head, or if there was, it hasn't been an issue in years since we haven't added new reserved prefixes in a long time.)

Reply:
I do not have a strong opinion on the issue, and I personally can live with the change. I was just noting that the change goes slightly beyond the editorial change of the PR.

It may be a bit of an overkill to raise this as a separate issue, though. It would be good to get another approval for the change before we go ahead, though. @reidmore-online ? @bduga ?

Reply:
How about as a compromise we strip the text about where the issues might occur and leave the example to explain, so only say:

> 
```suggestion
						<p>Although reserved prefixes are an authoring convenience, they can cause issues. Vendors, for example, will often reject new prefixes until they update their [=EPUB
```

If someone knows of another example, we can always add it.

Reply:
Works for me. Committing.